src/module-1/styles/css/style.css

```diff
@@ -0,0 +1,156 @@
+*,
```

Comment:
please consider having  
`global.css` for global styles
`list.css` for list styles
`card.css` for card  
names are examples, use whatever you like